docs/running-a-node/ledger-management.md

```diff
@@ -0,0 +1,141 @@
+# Ledger Management
+
+The node automatically manages the full Nano ledger in the `data.ldb` file which can be found in the data folder at these locations:
+
+--8<-- "folder-locations.md"
+
+This file will grow in size as the ledger does. As of April 2020 there are over 49 million blocks in the ledger which requires at least 26GB of free space. See [hardware recommendations](/running-a-node/node-setup/#hardware-recommendations) for more preferred node specs.
+
+!!! warning "RocksDB uses many files"
+	The above details are for the default LMDB database setup. If using RocksDB, please note that it uses potentially 100s of SST files to manage the ledger so details should followed from the [RocksDB Ledger Backend](#rocksDB-ledger-backend) section below.
+
+## Bootstrapping
+
+When starting a new node the ledger must be downloaded and kept updated in order to participate on the network properly. This is done automatically via bootstrapping - the node downloads and verifies blocks from other nodes across the network. This process can take hours to days to complete depending on network conditions and [hardware specifications](/running-a-node/node-setup/#hardware-recommendations).
```

Comment:
Would add here that the operator should avoid restarting the node unless it's very clear from calls to RPC block_count that both block counts are stuck for a long time. It simply delays the bootstrap further.

Reply:
Callout added